.circleci/config.yml

```diff
@@ -47,16 +55,17 @@ jobs:
             bin/build
             mkdir -p /tmp/docker/images
             docker save -o /tmp/docker/images/arnold.tar "${ARNOLD_IMAGE}"
-      # Store the built as an artifact that can be reused in other jobs
+      # Save built to the workspace so that it can be reused in other jobs
```

Comment:
s/built/build?

Reply:
I meant save **the** built to the workspace.